### PR TITLE
is0501: check that Content-Type of transportfile is correct

### PR DESCRIPTION
**Pull request by andrewbonney**
Adds a check for Content-Type returned by /transportfile, just checking SDP files for now.

### CODE REVIEW

File: nmostesting/suites/IS0501Test.py [hunk 1]
```diff
@@ -832,6 +832,42 @@ def test_41(self, test):
...
+                if valid:
+                    transport_type = response
+                else:
+                    return test.FAIL(response)
+
+            if transport_type == "urn:x-nmos:transport:rtp":
+                url = self.url + "single/senders/{}/transportfile".format(sender)
+                valid, response = self.do_request("GET", url)
+                if valid and response.status_code == 200:
+                    if response.headers["Content-Type"] != "application/sdp":
```
Comment (garethsb-sony) [inline nmostesting/suites/IS0501Test.py]: How about if no Content-Type header?

Comment (andrewbonney) [reply nmostesting/suites/IS0501Test.py]: Good point

File: nmostesting/suites/IS0501Test.py [hunk 2]
```diff
@@ -832,6 +832,42 @@ def test_41(self, test):
...
+                        return test.WARNING("RTP Sender {} failed to indicate a transportfile Content-Type of "
+                                            "application/sdp".format(sender))
+                elif valid and response.status_code == 404:
+                    access_error = True
+                else:
+                    return test.FAIL("Unexpected response from IS-05 API")
```
Comment (garethsb-sony) [inline nmostesting/suites/IS0501Test.py]: Think we call it Connection API in messages usually 🤷‍♂️

**Comment by garethsb-sony** [comment]
SDP files are largely represented in UTF-8, but the `a=charset` attribute allows a few lines to be interpreted differently. It's probably the case therefore that the `application/sdp` media type SHOULDN'T have a `charset` media type parameter, but how do you think we should handle this?

**Comment by andrewbonney** [comment]
Sure, I'd be happy to add a similar check here.

**Pull request merged**